src/main/java/edu/cornell/kfs/sys/document/authorization/WebServiceCredentialAuthorizer.java

```diff
@@ -0,0 +1,31 @@
+package edu.cornell.kfs.sys.document.authorization;
+
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.sys.document.authorization.FinancialSystemMaintenanceDocumentAuthorizerBase;
+import org.kuali.rice.core.api.parameter.ParameterEvaluator;
+import org.kuali.rice.core.api.parameter.ParameterEvaluatorService;
+import org.kuali.rice.kim.api.identity.Person;
+
+import edu.cornell.kfs.sys.CUKFSParameterKeyConstants;
+import edu.cornell.kfs.sys.businessobject.WebServiceCredential;
+
+public class WebServiceCredentialAuthorizer extends FinancialSystemMaintenanceDocumentAuthorizerBase {
+
+    @Override
+    public boolean canMaintain(Object dataObject, Person user) {
+        if (!super.canMaintain(dataObject, user)) {
+            return false;
+        }
+
+        WebServiceCredential webServiceCredential = (WebServiceCredential) dataObject;
+        ParameterEvaluator parameterEvaluator = SpringContext.getBean(ParameterEvaluatorService.class).getParameterEvaluator(
```

Comment:
I would move ParameterEvaluatorService into a getter and setter, and use spring conect in the getter in the local variable for ParameterEvaluatorService isn't set.